Updated the instructions to use the new Parter Integration Workflow (#4)
etleap/etleap-redshift-dev-days: README.md

We'll ETL data from two different data sources into your Redshift data warehouse. 
One source, an MongoDB source, contains JSON-formatted click event data. 
The other, a MySQL database, contains information about users. 

First, we need to setup our Redshift Cluster as a connection in Etleap.
For this setup you'll need the password you used when creating the Redshift Cluster.

We will set up the Redshift Connection using the Redshift Partner Integration:
- Go to the Redshift Console [here](https://console.aws.amazon.com/redshiftv2/home?region=us-east-1#clusters).
- Select the cluster your just created. 
- In the top right corner, click on the "Add Partner Integration" button.
- From the list of partners, select "Etleap". It should be the first option. Click "Next".
- Leave all the settings as they are and click "Add partner".
- This will take you to the Etleap console.
  - Confirm your Redshift password. Use the same password you used when you created the Redshift Cluster. Click "Validate and Setup Connection."
  - Add your email address. This will send you a confirmation email. Click the link in the email to continue.
- Fill in your details, and click "Create Account!"

> - Congratulation! Your account and connection are now ready to use. We'll go ahead, and set up more connections.

Reviewer: ```suggestion
- Congratulations! Your Etleap account and Redshift connection are now ready to use.
```

## 3 Ingesting data from MongoDB sources

In this section, we'll configure a MongoDB source connection, and create pipelines from it.

### 3.1 Set up the MongoDB connection

Use the search box to filter for "Mongo", and click on the MongoDB icon to create a new connection.
Use the following values for the inputs:

- Name: `Website Events`
- Hosts and Ports: `test.dev.etleap.com:27017`
- Database: `webstore`
- Authentication Database: `test` 
- Username: `devdays`
- Password: `i5eU2nZx4d`

> Let the 2 checkboxes unchecked.

Reviewer: ```suggestion
Leave the 2 checkboxes unchecked.
```

Click 'Save'. 

> This will take you to the list of available collection in the MongoDB database.

Reviewer: ```suggestion
This will take you to the list of available collections in the MongoDB database.
```

## 1. Set up a Redshift Cluster

In this section we'll set up a new Redshift cluster

Log into your AWS account and [go to the Redshift Console](https://console.aws.amazon.com/redshiftv2/home?region=us-east-1#clusters). 

- Make sure the AWS region selected is N. Virginia (us-east-1).
- Click "Create Cluster" in top right-hand corner.
- For the node type select `dc2.large`.
- For the number of nodes, select `1`.
- Under "Database configurations", enter a password that you'll remember. We will need this to connect to the cluster later on.
- Leave all other settings as they are.

Click "Create cluster".
It will take 5-10 minutes for the cluster to start up.

## 2. Connect Etleap to Redshift and the data sources, and set up pipelines

We'll ETL data from two different data sources into your Redshift data warehouse. One source, an SFTP source, contains JSON-formatted click event data. The other, a MySQL database, contains information about users. 

> The first thing you'll need to do is log into Etleap using the credentials that were provided to you via email. The email has the subject line 'Your Etleap Account'.

Reviewer: No longer correct, right?

Author: Correct. Removed.

In the rest of this section we'll connect Etleap to the data sources and Redshift destination, so that we can create pipelines in the next section.

> ### 2.1. Set up the Redshift connection via the Partner Integration Console

Reviewer: Would it make sense for this to be section 2 and the pipeline setup be section 3?

Author: I have made each source be its own section.
